python/cudf/cudf/_lib/includes/parquet.pxd

```diff
@@ -47,3 +49,49 @@ cdef extern from "cudf/cudf.h" namespace "cudf::io::parquet" nogil:
         cudf_table read_rows(size_t skip_rows, size_t num_rows) except +
 
         cudf_table read_row_group(size_t row_group) except +
+
+cdef extern from "cudf/io/functions.hpp" \
+        namespace "cudf::experimental::io" nogil:
+
+    ctypedef enum compression_type:
+        none "cudf::experimental::io::compression_type::NONE"
+        snappy "cudf::experimental::io::compression_type::SNAPPY"
+
+    ctypedef enum statistics_freq:
+        STATISTICS_NONE = 0
```

Comment:
Add full C++ name after this, I'm not sure if it just becomes an int in the generated cpp afterwards.